jbpm-designer-backend/src/main/java/org/jbpm/designer/bpmn2/validation/BPMN2SyntaxChecker.java

```diff
@@ -322,20 +324,52 @@ private void checkFlowNode(FlowNode flowNode, FlowElementsContainer container, P
 	}
 
 	private void checkBusinessRuleTask(BusinessRuleTask businessRuleTask) {
```

Comment:
@mswiderski I had look also on the linked PR for jbpm. I see that there is done pretty same kind of validation like in this method. And there come idea to my mind. Isn't it possible to replace this method with something like [1]? Would be there many problems with error I18N and binding to specific node? What do you think @mswiderski @tsurdilo ?

[1]
https://github.com/mswiderski/jbpm-designer/blob/544bbd2f3b68a1826253f9468d215daa67d63a15/jbpm-designer-backend/src/main/java/org/jbpm/designer/bpmn2/validation/BPMN2SyntaxChecker.java#L152